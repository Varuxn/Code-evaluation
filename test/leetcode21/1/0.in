100 100
k
zhehxrtysjiyqkstqesx
fhwdwsyqplupkzbotphsmuaudqpmd
hjlfixwypbvztvatjjoyfjzrvxze
pfbvrjynokzdlatsafirigjximjqguxgiqxktxsjbddrsyej
mn
xruhngrbovhln
igjqfmubqgyvleriqavrbntfttfawsbaffreyqghy
foaphjduvittafoujf
bzxwngmuaptfpofifwm
jemqhvhixzbejuuyzseatgewnrscsyxmacqgrwtj
kgiiuovoxtdbqakizuwlrcwlb
hfau
zbkqgigkrxclcdg
yentbeqbbo
feilvsdwzvsoqzfcjvjqtovbxautmdcogxdvknbph
jailexvhh
asvqhgcsjzkkysozwrbnbuy
phjmdxexrzipwnxrnundhcdudy
mzsogyknyzqswg
peeqfqswmvbcwuxafouae
wathxmlvupwgwyjvhdkjlynqxwahn
fmwyynptvalrgrfnaiquzvrukujhukpflalvlbic
dxojrbyvbhnddxchurjusjhmivwartldrurj
nammzvdjfqifdufxmjlk
tdchvkmiqjbpbkbb
hsskohjfytfmgmqvqgis
ksstwysysvjodmlfa
wesgogefmjpofbczojzwakozi
xkiesxehysi
qiczktbxkuplvqfyukiwieyrbxelfbidptukbmiyqsr
ddxtjkjbszvyubtuihqtyasybmepbhzgfzmjureqjmlhmvfgyn
nxfjuruuixpptslwgzgfjhmgwpoyvcyf
rptwvavoapphdqprgncezrczkqaecl
jgxjqdiqzgctkactjebbgxdc
pucqrobohimztqxjyxsdidyzqntpbqfmbsquuohwfvrze
owkitgjgbjigwrnimedqnkaoznjr
oiiiygwdqenvufabnujrxpzdjqoesevudrpvjar
kqfpxerhdnmdagygrrueadkuqqcnqeqyiyhou
mktwayolwhrlzcvozajtgrwcyfpdtc
pauafdgnyetzvqohlmkuxdstakptcgnurningcvqmoctlydtf
owolvllkhpuxnweqwgozlwubljyahrrehr
vfiocuxnfuzaw
ascllwanorqpkjzyrlxpcstgkgxcsskjxlfdyrebms
bzvhbwsr
goz
sxfdmeodxoxhghiskqaytdtkrrfidwmhzi
tsrdkhzykafsuljrdwjtbf
ndk
jxxn
ajlnwspgsrivdtxnukpgukpfetokbbmrxpbpumtfvlqwgucp
zvuzaurihuxhhcgdsqdwmugc
llspctrnrglmue
lbpuktydaqnejweidoxqznc
dxisftgqgjiijyowwpbdjskbehuvinlckngrjgmect
sko
fflqoeisnfuvxwwvelptuabryueplderiahqclvgpixrmlcw
azdczpnpkitajwnmokkynhnczrsmsitccinh
sguef
ycroaytlcoelpcjatufpqwbkelbtxrjonerm
dzefwarhr
benywckviwgljtpzkt
wdwyvpfz
xkaxjoaimm
qkujdwyxhnxe
fvxyazwqkk
osrjpibngdccqlaslkauboxnhpqsg
iazokaapntemk
lzwd
ebqacegoycgwygbglxtzxplnyvfogokuuzhfdywmemnsslbi
upmjmiaiwgrz
wbafuo
multvbajglh
edqa
eoxhiedeemdjqxqbuoeqejwmeitg
wevaqp
dlzdlkexbujkbnakqzzafglahuiey
pyuzvoibknyz
vnkizikc
xjdbofimkcygc
dvihqbxromeosvntarugiijpzjlkhywbyaochsv
qcfrexcmqael
nwansajfndslxzjkhfnwgkmxp
rdnwnplyxapksbpona
dokphjrjqjwbexajajlijpxvdcmmtxpjtelvxxdjefdezllqt
wssakqwely
cgvpvlpniauqnpqjlynntigwzpvfhxticlyxqlhktzwt
yhnagjboxwjznrjbmdrxtgmfoyeicxamlctp
wwqrgdekwdqjbzojngxkqgqivvi
kuxdgycdnzuiqrzkfjo
zmwcsyxdhgydffkccjy
vwtovcgnjfqdlyuufaisprgdzbwxfobxekxpieemtaquobuv
rvcozkersrqroloiumumyknansnojujnhcnndxa
lkzsygpfmpqvydfgwohjfswselymklzuu
spnotdyh
wrzxhmfdvsamyfnwevfjzeguuihytrqrsaobwepxsc
dbfmmcvpscqfocbfeomiwiyuwvdzuvygghzdvbyshhkkzfvpd
pmeaplzzzncsfudlsuvuhooslzwgdrdjc
ecbdxfwbpnztziqqoxevginwnwrkn
zkaobotvgzz
qdpkpqdlbsming kqfpxerhdnmdagygrrueadkuqqcnqeqyiyhouing peeqfqswmvbcwuxafouaeing ikdoogbvjwmjdvfruvshgolqltvojvbcggsltxqwerqwe ixgnnqmadwkowhkkhbmfstpokrf qcfrexcmqaeled rqlcyzxjylydobgzeepelslaamfkwvzzxbrluznloing dokphjrjqjwbexajajlijpxvdcmmtxpjtelvxxdjefdezllqting nwansajfndslxzjkhfnwgkmxping nxfjuruuixpptslwgzgfjhmgwpoyvcyfed vjbrphdptwdoxqnbcaed oiiiygwdqenvufabnujrxpzdjqoesevudrpvjaring wnrhptjedpiohfkafrywzcing rvcozkersrqroloiumumyknansnojujnhcnndxaing kpnqetsybed gozing kgiiuovoxtdbqakizuwlrcwlbing qkujdwyxhnxeed peeqfqswmvbcwuxafouaeing ordwqtofjpfohwades kqfpxerhdnmdagygrrueadkuqqcnqeqyiyhouing tdknjpyquigvqaizbmgynagmoesjnnoriadipqtwed bzxwngmuaptfpofifwmed asvqhgcsjzkkysozwrbnbuying vfiocuxnfuzawes yedkugmefghzsmhxtydzvwqil skoing pjhing dvihqbxromeosvntarugiijpzjlkhywbyaochsvqwerqwe fjywesmrehprbawtaing feilvsdwzvsoqzfcjvjqtovbxautmdcogxdvknbphed rvcozkersrqroloiumumyknansnojujnhcnndxaing fvxyazwqkkqwerqwe jailexvhhing oqrhusfvfuewlqfibokxtluwmzaqxwmrggqwgpkofhed eoxhiedeemdjqxqbuoeqejwmeitg tzuegkxeatmvfttxxiyligtacmfchoxzhqdvelukdrs dswmyyfexfystobrimlning nammzvdjfqifdufxmjlked azoavfgfxvojxagrsocyalvwusybuvrsving kgiiuovoxtdbqakizuwlrcwlbing kgspijrcsynilntmxizzyucieydzpjbalovwpdvstgcxkunabing yqudiyzyoxktmnanvqomaveltoemyzcugedcepbpnied ndkes pmeaplzzzncsfudlsuvuhooslzwgdrdjces cibxxxlnfqumqazymawdxueuyplyiuving cgvpvlpniauqnpqjlynntigwzpvfhxticlyxqlhktzwting mn kqfpxerhdnmdagygrrueadkuqqcnqeqyiyhoues jxxn ndedfmhmmnvogrumrtixyhyjklxkovpofming pyuzvoibknyzed lsxacadadqhmtuzgctwdwrkgonxvoznulufed nammzvdjfqifdufxmjlkes ikvqysrasring kbrobing lbpuktydaqnejweidoxqzncing ncxfrvygurdnwprhfbekmlrdfbciaruwzcwdemxdeszeeded eoxhiedeemdjqxqbuoeqejwmeitging nxfjuruuixpptslwgzgfjhmgwpoyvcyfing umzdtfeed eoxhiedeemdjqxqbuoeqejwmeitged dokphjrjqjwbexajajlijpxvdcmmtxpjtelvxxdjefdezllqting benywckviwgljtpzkting imdaing tkhpyhckfieaphbjqhwfwlryfqaciqoriszfkjuing hsskohjfytfmgmqvqgising urfcquyvuudofynhtqwerqwe skoes zxcbejuvaiyryicncwxees qkujdwyxhnxeed zjfing bzxwngmuaptfpofifwmed jwtrsscxnrqhtsoflqbdmxdgring rptwvavoapphdqprgncezrczkqaecling odjfoskvjdxhlvylxuxboqmed cspbpawving ascllwanorqpkjzyrlxpcstgkgxcsskjxlfdyrebmsing msifpvnkxqyzyjpjsvkigmevkfkhmcazktysrszlred azdczpnpkitajwnmokkynhnczrsmsitccinhing jegppfhvuzfwtpbidiqxhzhtulfes vtihfboaes wesgogefmjpofbczojzwakoziing skoing nqwerqwe ijkzoduhtfuxzaamxgnhing pfbvrjynokzdlatsafirigjximjqguxgiqxktxsjbddrsyejing gozed peeqfqswmvbcwuxafouaeed kunuhggvzhwxsrweanhbhrwred pmeaplzzzncsfudlsuvuhooslzwgdrdjc zvuzaurihuxhhcgdsqdwmugcing orvaubing nowjannhdkkkkaiglsxtqlkrwbyogqzhing wssakqwelying hjlfixwypbvztvatjjoyfjzrvxzeing dxowywokmojkdkwwkvhbmbhnykxeing pucqrobohimztqxjyxsdidyzqntpbqfmbsquuohwfvrzeing xqodxefsrdorbxnndxrureoaprndzfcfnlxkuthqed tsrdkhzykafsuljrdwjtbfing
