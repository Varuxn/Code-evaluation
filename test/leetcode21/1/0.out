qdpkpqdlbsming k peeqfqswmvbcwuxafouae ikdoogbvjwmjdvfruvshgolqltvojvbcggsltxqwerqwe ixgnnqmadwkowhkkhbmfstpokrf qcfrexcmqael rqlcyzxjylydobgzeepelslaamfkwvzzxbrluznloing dokphjrjqjwbexajajlijpxvdcmmtxpjtelvxxdjefdezllqt nwansajfndslxzjkhfnwgkmxp nxfjuruuixpptslwgzgfjhmgwpoyvcyf vjbrphdptwdoxqnbcaed oiiiygwdqenvufabnujrxpzdjqoesevudrpvjar wnrhptjedpiohfkafrywzcing rvcozkersrqroloiumumyknansnojujnhcnndxa k goz k qkujdwyxhnxe peeqfqswmvbcwuxafouae ordwqtofjpfohwades k tdknjpyquigvqaizbmgynagmoesjnnoriadipqtwed bzxwngmuaptfpofifwm asvqhgcsjzkkysozwrbnbuy vfiocuxnfuzaw yedkugmefghzsmhxtydzvwqil sko pjhing dvihqbxromeosvntarugiijpzjlkhywbyaochsv fjywesmrehprbawtaing feilvsdwzvsoqzfcjvjqtovbxautmdcogxdvknbph rvcozkersrqroloiumumyknansnojujnhcnndxa fvxyazwqkk jailexvhh oqrhusfvfuewlqfibokxtluwmzaqxwmrggqwgpkofhed eoxhiedeemdjqxqbuoeqejwmeitg tzuegkxeatmvfttxxiyligtacmfchoxzhqdvelukdrs dswmyyfexfystobrimlning nammzvdjfqifdufxmjlk azoavfgfxvojxagrsocyalvwusybuvrsving k k yqudiyzyoxktmnanvqomaveltoemyzcugedcepbpnied ndk pmeaplzzzncsfudlsuvuhooslzwgdrdjc cibxxxlnfqumqazymawdxueuyplyiuving cgvpvlpniauqnpqjlynntigwzpvfhxticlyxqlhktzwt mn k jxxn ndedfmhmmnvogrumrtixyhyjklxkovpofming pyuzvoibknyz lsxacadadqhmtuzgctwdwrkgonxvoznulufed nammzvdjfqifdufxmjlk ikvqysrasring k lbpuktydaqnejweidoxqznc ncxfrvygurdnwprhfbekmlrdfbciaruwzcwdemxdeszeeded eoxhiedeemdjqxqbuoeqejwmeitg nxfjuruuixpptslwgzgfjhmgwpoyvcyf umzdtfeed eoxhiedeemdjqxqbuoeqejwmeitg dokphjrjqjwbexajajlijpxvdcmmtxpjtelvxxdjefdezllqt benywckviwgljtpzkt imdaing tkhpyhckfieaphbjqhwfwlryfqaciqoriszfkjuing hsskohjfytfmgmqvqgis urfcquyvuudofynhtqwerqwe sko zxcbejuvaiyryicncwxees qkujdwyxhnxe zjfing bzxwngmuaptfpofifwm jwtrsscxnrqhtsoflqbdmxdgring rptwvavoapphdqprgncezrczkqaecl odjfoskvjdxhlvylxuxboqmed cspbpawving ascllwanorqpkjzyrlxpcstgkgxcsskjxlfdyrebms msifpvnkxqyzyjpjsvkigmevkfkhmcazktysrszlred azdczpnpkitajwnmokkynhnczrsmsitccinh jegppfhvuzfwtpbidiqxhzhtulfes vtihfboaes wesgogefmjpofbczojzwakozi sko nqwerqwe ijkzoduhtfuxzaamxgnhing pfbvrjynokzdlatsafirigjximjqguxgiqxktxsjbddrsyej goz peeqfqswmvbcwuxafouae k pmeaplzzzncsfudlsuvuhooslzwgdrdjc zvuzaurihuxhhcgdsqdwmugc orvaubing nowjannhdkkkkaiglsxtqlkrwbyogqzhing wssakqwely hjlfixwypbvztvatjjoyfjzrvxze dxowywokmojkdkwwkvhbmbhnykxeing pucqrobohimztqxjyxsdidyzqntpbqfmbsquuohwfvrze xqodxefsrdorbxnndxrureoaprndzfcfnlxkuthqed tsrdkhzykafsuljrdwjtbf